app/forms/steps/outgoings/council_tax_form.rb

```diff
@@ -0,0 +1,40 @@
+module Steps
+  module Outgoings
+    class CouncilTaxForm < Steps::BaseFormObject
+      include Steps::HasOneAssociation
+      has_one_association :outgoings
+
+      attribute :pays_council_tax, :value_object, source: YesNoAnswer
+      attribute :council_tax_amount, :integer
+
+      validates_inclusion_of :pays_council_tax, in: :choices
+
+      validates :council_tax_amount,
```

Comment:
One thing you can do here is say 'Yes' to paying council tax and enter zero and the form submits, thinking that may be good to have a validation that the value entered needs to be > 0? 🤔

Reply:
👍 Now negative or 0 doesn't pass validation and shows error message